kalite/coachreports/static/js/coachreports/tabular_reports/hbtemplates/tabular-view.handlebars

```diff
@@ -1,7 +1,7 @@
 <div>
     <div class="row">
         <div class="col-md-6 col-xs-12">
-        <label id="legendMargin" for="legend">Key</label>
+        <label id="legendMargin" for="legend">{{_ "Color legend"}}</label>
```

Comment:
@benjaoming I'm not sure if this is the best solution, I would keep the string `Key`, it's simpler and already present on the page:

![color-legend](https://user-images.githubusercontent.com/1457929/28972811-69a831ec-7931-11e7-9bee-9754bfe9dee0.png)